app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,58 @@
     font-weight: normal;
   }
 }
+
+.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  
+  li {
+    display: inline-block;
+    height: 150px;
+    margin: 100px 1em 1em 0;
+    position: relative;
+    width: 150px;
+    
+    &:hover span.text-content {
+      opacity: 1;
+    }
+  }
+}
+
+.text-content {
+  background: rgba(0, 0, 0, 0.5);
+  color: $blue;
+  cursor: pointer;
+  display: table;
+  height: 100px;
+  left: 0;
+  opacity: 0;
+  position: absolute;
+  top: 0;
+  -moz-transition: opacity 500ms;
+  -o-transition: opacity 500ms;
```

Comment:
you should be able to use `transition` from bourbon: http://bourbon.io/docs/#transitions

something like:
```scss
@include transition(opacity 500ms);
```